apps/server/src/shared/domain/entity/task.entity.ts

```diff
@@ -75,12 +76,15 @@ export class Task extends BaseEntityWithTimestamps implements ILearnroomElement,
 	@Property()
 	private = true;
 
+	@Property({ nullable: true })
+	publicSubmissions?: boolean;
+
 	@Index()
 	@ManyToOne('User', { fieldName: 'teacherId', nullable: true })
 	creator?: User;
 
 	@Index()
-	@ManyToOne('Course', { fieldName: 'courseId', nullable: true })
+	@ManyToOne('Course', { fieldName: 'courseId', nullable: true, eager: true })
```

Comment:
We must really avoid eager loadings on any place. It is to dangour.

Reply:
see above